packages/jam/jam-host-calls/externalities/partial-state.ts

```diff
@@ -231,6 +233,7 @@ export interface PartialState {
     gas: ServiceGas,
     allowance: ServiceGas,
     gratisStorage: U64,
+    serviceId: U64,
```

Comment:
```suggestion
    requestedServiceId: U64,
```
or `wantedServiceId`.

Would be good to update the docs as well.